CODEOWNERS

```diff
@@ -13,13 +13,19 @@
 */equilibria/ @Fusion-Power-Plant-Framework/equilibria-review
 */fuel_cycle/ @Fusion-Power-Plant-Framework/fuel-cycle-review
 */geometry/ @Fusion-Power-Plant-Framework/geometry-review
-*/magnetostatics/@Fusion-Power-Plant-Framework/magnetostatics-review
+*/magnetostatics/ @Fusion-Power-Plant-Framework/magnetostatics-review
 */materials/ @Fusion-Power-Plant-Framework/materials-review
 */mesh/ @Fusion-Power-Plant-Framework/mesh-review
 */balance_of_plant/ @Fusion-Power-Plant-Framework/power-cycle-review
 */radiation_transport/ @Fusion-Power-Plant-Framework/radiation-transport-review
 */utilities/  @Fusion-Power-Plant-Framework/utilities-review
 
+# Repository responsibility
+
+*/ @Fusion-Power-Plant-Framework/repo-review
```

Comment:
I'm intending this to hit everything in the top level directory but NOT sub-directories. Unfortuantely I can't know if this works a priori... @je-cook maybe you know

Reply:
I'm pretty sure this is wrong